.github/workflows/utils/workflows.json

```diff
@@ -94,5 +94,6 @@
   "video-intelligence",
   "vision/productSearch",
   "workflows",
-  "workflows/invoke-private-endpoint"
+  "workflows/invoke-private-endpoint",
+  "security-center/snippets/v2"
```

Comment:
issue: #3883 means we shouldn't separately add v2.

Reply:
Addressed.